extensions/arrow/src/test/java/io/deephaven/extensions/arrow/ArrowVarCharVectorTest.java

```diff
@@ -34,7 +34,7 @@ public class ArrowVarCharVectorTest {
     private static QueryTable loadTable() {
         //noinspection ConstantConditions;
         final File dataFile = new File(ArrowVarCharVectorTest.class.getResource("/var_char_vector.arrow").getFile());
-        return ArrowWrapperTools.readArrow(dataFile.getPath());
+        return ArrowWrapperTools.readFeather(dataFile.getPath());
```

Comment:
Can you find test data and confirm if we can handle FeatherV1, FeatherV2, or both?